homeassistant/components/wled/config_flow.py

```diff
@@ -31,27 +31,27 @@ async def async_step_user(
         return await self._handle_config_flow(user_input)
 
     async def async_step_zeroconf(
-        self, user_input: ConfigType | None = None
+        self, discovery_info: ConfigType | None = None
```

Comment:
The discovery info should never be `None`.

Reply:
We can address this in a follow up too if wanted.

Reply:
Yes, I know. The typing in that component is already pretty broken and it's not checked anyway. So I think right now it's not worth fixing it before we can enable type checks here. That I'm trying to do in https://github.com/home-assistant/core/pull/49270

Reply:
But yeah, let's annotate this method correctly at least.

Reply:
Ok, let's fix this separately then.